cypress/e2e/timeline/timeline.spec.ts

```diff
@@ -360,5 +360,24 @@ describe("Timeline", () => {
             cy.get(".mx_RoomView_body .mx_EventTile > .mx_EventTile_line > .mx_MVoiceMessageBody")
                 .should("have.length", 1);
         });
+
+        it("should not be possible to send flag with regional emojis", () => {
+            cy.visit("/#/room/" + roomId);
+
+            // Send a message
+            cy.getComposer().type(`:regional_indicator_a`);
+            cy.wait(300);
+            cy.getComposer().type(`{enter}:regional_indicator_r`);
+            cy.wait(300);
+            cy.getComposer().type(`{enter} :regional_indicator_z`);
+            cy.wait(300);
+            cy.getComposer().type(`{enter}:regional_indicator_a`);
+            cy.wait(300);
+            cy.getComposer().type(`{enter}{enter}`);
```

Comment:
```suggestion
            cy.getComposer().type(":regional_indicator_a");
            cy.contains(".mx_Autocomplete_Completion_title", ":regional_indicator_a:").click();
            cy.getComposer().type(":regional_indicator_r");
            cy.contains(".mx_Autocomplete_Completion_title", ":regional_indicator_r:").click();
            cy.getComposer().type(" :regional_indicator_z");
            cy.contains(".mx_Autocomplete_Completion_title", ":regional_indicator_z:").click();
            cy.getComposer().type(":regional_indicator_a");
            cy.contains(".mx_Autocomplete_Completion_title", ":regional_indicator_a:").click();
            cy.getComposer().type("{enter}");
```
Something along these lines is a better way to wait for the emoji picker to load, rather than waiting a fixed span of time even if it shows up sooner. Also <code>`</code> should only be used for template literals.